py-geopolars/src/geoseries.rs

```diff
@@ -141,6 +141,19 @@ pub(crate) fn skew(
     ffi::rust_series_to_py_geoseries(&out)
 }
 
+#[pyfunction]
+pub(crate) fn distance(
+    series: &PyAny,
+    other: &PyAny,
+) -> PyResult<PyObject> {
+    let series = ffi::py_series_to_rust_series(series)?;
+    let other = ffi::py_series_to_rust_series(other)?;
+    let out = series
+        .distance(&other)
+        .map_err(PyGeopolarsError::from)?;
+    ffi::rust_series_to_py_geoseries(&out)
```

Comment:
This should be `rust_series_to_py_series` I believe, as the return values is non-geographic